repobee_sanitizer/_sanitize_repo.py

```diff
@@ -64,7 +66,7 @@ def discover_dirty_files(
 
 def sanitize_files(
     basedir: pathlib.Path, file_relpaths: List[_fileutils.RelativePath]
-) -> List[_format.FileWithErrors]:
+):
```

Comment:
Omitting the type hint is not the same as stating that it returns `None`, as type hints themselves are optional. Omitting the type hint implicitly makes the type [`Any`](https://www.python.org/dev/peps/pep-0484/#the-any-type), which can be substituted for _any_ type.

```suggestion
) -> None:
```